README.md

```diff
@@ -1 +1,228 @@
-# ursa-rfcs
\ No newline at end of file
+# Ursa RFCs
+[Ursa RFCs]: #ursa-rfcs
+
+Many changes, including bug fixes and documentation improvements can be
+implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put through a
+bit of a design process and produce a consensus among the Ursa community.
+
+The "RFC" (request for comments) process is intended to provide a consistent
+and controlled path for major changes to enter Ursa and other official
+project components, so that all stakeholders can be confident about the
+direction Ursa is evolving in.
+
+This process is intended to be substantially similar to the Rust RFCs process,
+customized as necessary for use with Ursa. The README.md and
+0000-template.md were initially forked from [Rust
+RFCs](https://github.com/rust-lang/rfcs).
+
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+  - [Opening](#ursa-rfcs)
+  - [Table of Contents]
+  - [When you need to follow this process]
+  - [Before creating an RFC]
+  - [What the process is]
+  - [The RFC life-cycle]
+  - [Reviewing RFCs]
+  - [Implementing an RFC]
+  - [Help this is all too informal!]
+  - [License]
+
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make "substantial" changes to
+Ursa or any of its sub-components or the  RFC process itself. What constitutes a
+"substantial" change is evolving based on community norms and varies depending
+on what part of the project you are proposing to change, but may include the
+following.
+
+  - Architectural changes
+  - Substantial changes to component interfaces
+  - New core features
+  - Backward incompatible changes
+  - Changes that affect the security of communications or administration
+
+Some changes do not require an RFC:
+
+  - Rephrasing, reorganizing, refactoring, or otherwise "changing shape does
+    not change meaning".
+  - Additions that strictly improve objective, numerical quality criteria
+    (warning removal, speedup, better platform coverage, more parallelism, trap
+    more errors, etc.)
```

Comment:
Can we provide some examples here that are crypto-related for guidance?  For instance, proposing a new protocol definitely should require an RFC.  But what about a new implementation of an existing protocol (that could be very different from the old implementation, perhaps with different security properties)?

Reply:
Will update with examples including that new implementations of previously approved algorithms require RFCs. All RFCs will include the motivation for the request, i.e. what is the upstream requirement.

Reply:
Saw the change.  Thanks!  I hope as the RFCs come in, we can add some more concrete examples.